src/main/java/com/razorpay/PaymentClient.java

```diff
@@ -122,4 +122,7 @@ public Payment validateUpi(JSONObject request) throws RazorpayException {
   public Payment expandedDetails(String id, JSONObject request) throws RazorpayException {
     return get(Constants.VERSION, String.format(Constants.PAYMENT_GET, id), request);
   }
+  public Methods fetchPaymentMethods() throws RazorpayException {
+    return get(Constants.VERSION, Constants.FETCH_PAYMENT_METHODS, null);
```

Comment:
Please rename constant FETCH_PAYMENT_METHODS to METHODS

Reply:
renamed constant `FETCH_PAYMENT_METHODS` to `METHODS `